manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Casey's Extension",
+    "version": "1",
+    "description": "Hello World!",
+    "permissions": ["activeTab"],
+    "manifest_version": 2,
+    "browser_action": {
+        "default_popup": "popup.html",
+        "default_icon": "casey_project.png",
```

Comment:
Nice icon ~
It's good practice to give your image (and other) files names that describe what they are and/or what purpose they serve. So here, a better name would be something like `browserIcon32.png`.
Also, it's good practice to keep your project resources organized into folders by type. Really helps things from getting out of hand disorganized once a project starts getting any kind of sizeable. Go ahead and make an `images` folder, chuck the icon file in there, and update the path here in the manifest accordingly. (Look at our extension manifest for an example if you need to).